src/main.ts

```diff
@@ -50,10 +50,11 @@ async function run() {
     }
     await rustup.installToolchain(opts.name, installOptions);
 
-    if (opts.target) {
-        await rustup.addTarget(opts.target, opts.name);
-    }
-
+	if (opts.targets) {
+		for (let target of opts.targets) {
+			await rustup.addTarget(target, opts.name);
```

Comment:
Maybe it would be possible to install them in parallel (`Promise.all`)?

Reply:
Do you know if it's safe to run `rustup` in parallel? I don't know really if there's any locking or global state...

Reply:
Nope, rustup is not entirely safe to run concurrently: https://github.com/rust-lang/rustup/issues/988

Reply:
In a quick test I was able to add two different components at the same time, but not remove them.